Course/IOS15(P1)/Lesson1/LessonView1.swift

```diff
@@ -0,0 +1,63 @@
+//
+//  Lesson1VIew.swift
+//  Course
+//
+//  Created by Марк Горкій on 11.10.2022.
+//
+
+import Foundation
+import SwiftUI
+
+struct ContentViewL1: View {
```

Comment:
Instead of creating `ContentViewL1` create a view, which accepts a model, which describes the course and render the view depending on it